### DISCUSSION TITLE
🔗 Ability to disable features (Announcement Popups, Elementor AI) (Duplicate of #22853)

### DISCUSSION THREAD

**Discussion by valmirselmani** [elementor/elementor#22796] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I would like to have the ability to disable certain features, in this case now the announcement popups and Elementor AI.
It confuses our clients, some of which aren't very technical.

I have already received several calls about what these popups mean and why AI appears everywhere in the editor.

**Announcement Popups**
The "Activate Containers for Brilliant Layouts" popup is displayed even for users who have no rights to change Elementor settings.
![Bildschirmfoto 2023-06-16 um 14 41 17](https://github.com/elementor/elementor/assets/1654866/6b56b9fc-f4e1-4de4-9ded-eb5a7e25167c)

### Describe the solution you'd like

Add two more controls for Announcements and Elementor AI to the Features tab in Elementor Settings. You can leave it enabled by default as long as I have the option to disable it.

### Describe alternatives you've considered

If it's too much work to do this in the UI, at least give us the option to disable these features via WordPress hooks and filters.

**Comment by robaxx** [comment]
Of course these things must be able to be controlled. On my multisite WP it's already bad enough, but Imagine running an intranet with WordPress publishing (as I once did) and these things are automatically rolled out to my users by Elementor without any control over them. At a minimum, put them in the role permissions feature so they can be disabled by default for non-admin roles and enabled as needed. It's already too late here though.

**Comment by marks7777** [comment]
YESS!

**Comment by rwkyyy** [comment]
God please make this a must for future releases, same for Apps!

**Comment by ronnyadsetts** [comment]
+1 Yup. Came here, annoyed by AI links everywhere and having to support this on the Elementor Pro websites we build. Time to look for an alternative page builder solution? Maybe the only way Elementor will listen to paying customers is if we stop paying?

**Comment by black-eye** [comment]
Yes, please. In addition to mentioned above, there are new nags for their Image optimization plugin (which shows even if other plugin for this purpose is installed). 

The same goes for the features which went to the core and is impossible to disable them, like Notes in the last version.

**Comment by mcrowell** [comment]
+1 - Please give us agency owners a way to disable these popups.

**Reply by ghost** [reply]
Agree. AI for now is useless thing. Better if they include some new functions.. like NOTES. 
Or to integrate FEP (floating elementor panel) cause this is useful.

**Comment by ghost** [comment]
+1 Unless you want more and more agency owners (like myself) to think about better alternatives (i.e., Elementor competitors), more controls need to be made available.

**Comment by awebguy99** [comment]
Yes, these popup ads (AI, Image Optimization etc) are becoming ridiculous, especially for an Elementor PRO subscriber!
Elementor devs, you need to add an option disable, even if it's just for Pro sites.

**Comment by robaxx** [comment]
Imagine if one had this installed on an Intranet... it shows a serious lack of understanding IT and business.

**Comment by MogulChris** [comment]
I also find this advertising intrusive. You don't even own the content curled out by the (pretty bad) AI features.

**Comment by c2wd** [comment]
I can't upvote this enough. Really getting tired of the annoying AI pop-ups everywhere. Just noticed another one the other day when I created a new page and added a new container "ELEMENTOR AI - Give your workflow a boost. Build containers with AI and generate..."

Enough already with the AI push!

**Comment by memdigital** [comment]
Yes, for advanced users managing our own client builds, I'd like to be able to remove all these ads and popups. I don't need to see them.

**Comment by squadcar** [comment]
These two functions will remove the vast majority of the AI prompts and controls (from @perks98 & @c2wd over [here](https://github.com/orgs/elementor/discussions/22853))

```
function remove_ai_control( $element, $section_id, $args ) {

	foreach( $element->get_controls() as $control ) {
		$element->update_control( $control['name'], [ 'ai' => false ] );
	}
}
add_action( 'elementor/element/before_section_start', 'remove_ai_control', 10, 3 );

function remove_elementor_ai_script_from_editor() {
	if ( \Elementor\Plugin::$instance->editor->is_edit_mode() ) {
        	wp_dequeue_script( 'elementor-ai' );
    	}
}
add_action( 'elementor/editor/after_enqueue_scripts', 'remove_elementor_ai_script_from_editor' );

```

**Reply by Thelastpoet** [reply]
@squadcar can this be used to remove any other controls and sections on the Advanced Tab?

**Reply by squadcar** [reply]
The first one yes, although I'd check the developer docs to make sure you're not messing up any dependancies. That said, it's not my code, I just copied and pasted it :)

**Reply by muzKore** [reply]
Would be great to make a plugin from some of these scripts. "Elementor Nags be gone".

**Comment by mikedoyledesign** [comment]
Just echoing my agreement here. PRO is not the place for this - we are and have been paying for a lean website builder. Offer upgrades on your own website dashboard or on elementor FREE, but leave them out of PRO.

**Comment by nicholaszein** [comment]
This Discussion was referenced in the following issues and discussions:

**Issues**
- #25739
- #25792
**Discussions**
- #22853
- #23263 
- #27354

**Reply by ghost** [reply]
Which comments are you referring to? This thread does not contain anything offensive or disrespectful. If you think that is the case, I'd recommend Elementor find others to oversee the posts here because the sensitivity level is too high. People could have been way more explicit if they could say how they really felt. 

I've been thinking about publishing a video on this topic because the way it has been handled is not good. Now, I'm definitely going to do the video just out of principle.

If you want to block me, be my guest. Contributing here is fruitless anyway.

**Reply by nicholaszein** [reply]
@wpturnedup the warning was posted after I had to delete very offensive and aggressive comments. That's why you didn't see them. 🫤

**Comment by nicholaszein** [comment]
_A word of advice, and a warning: Offensive or Disrespectful remarks, or any behavior that does not align with the [Code of Conduct](https://elemn.to/gh-code-of-conduct) will grant the author a block from contributing or participating in this repository. Please, be respectful. Keep the conversation amicable and civilized. I have no pleasure in having to block users. I am a user myself. But I will do it if I see any comment breaking the Code of Conduct._

**Comment by muzKore** [comment]
Can we please have new features added to the feature page (like AI has) and if I disable it, disable all of the nags and buttons. That makes sense and I have no idea why I continue to see nags. 
If this is not done done by Elementor, maybe someone can create a plugin that can be installed that will remove all of the annoyances that we should not have to put up with.

**Comment by onotype** [comment]
This whole thing with AI is a little bit tone deaf. We all know the reception of AI in the general public. We can all enjoy it as devs but I honestly don't have the energy or the patience to defend or explain to clients why their websites "have AI". If it was a setting I can turn off and forget, this would be a non issue. But no, it's not.

Fortunately, I haven't encountered a client adamantly against AI enough that I'd lose them for just having this. But is that really so hard to think about? What's elementor's recommendation for situations like that? Do we just grit our teeth and lose a client because of a plugin I paid for thought "you definitely need AI"?

Image optimization: If you're going to plaster ads on my client's website and not give me an option to turn it off, at least make it intelligent enough to recognize that other image optimization solutions exist. I use different solutions like Smush or TinyPNG, or in some cases, my managed hosting takes care of it for me. I have no need for this feature, and have assured my clients that images are optimized. But now I have to explain this whole thing to them and reassure them again.

What's Elementor's recommendation for these scenarios?

**Reply by black-eye** [reply]
My recent impression is that Elementor is not considering real proffesional developers and agencies their main target audience any more.  Othervice they wouldn't ruin our reputation in so many aspects. Elementor, please prove me wrong and give us an option to disable all this madness - it's ruining your reputation too!

Because, wheather intentional or not, all these aggressive ads, nags and notices have exactly this effect. What problem would it be for Elementor dev to make all these "features" and recommendations optional? Make it a special option at least for the highest tier (what a great incentive to upgrade, would it be... ;-) )

**Reply by mbaierl** [reply]
Would be easy to have a good practice and add some filters to allow to disable these features. No need for a checkbox in the UI... so only DEVs and agencies will disable these ads in a paid (!) plugin.

**Reply by toineenzo** [reply]
I recently had a call with someone working at Elementor. I also told him about these notices/ads and how it doesn't make you feel like a pro user anymore. He exactly shared the same opinion, and he said he would discuss all my things with their theme. Let's hope they change something about this, even if you can only disable the notices via a filter.

**Reply by squadcar** [reply]
> Would be easy to have a good practice and add some filters to allow to disable these features. No need for a checkbox in the UI... so only DEVs and agencies will disable these ads in a paid (!) plugin.

For the record, you can disable these things via functions/plugin/etc. [Scroll up a bit](https://github.com/orgs/elementor/discussions/22796#discussioncomment-9322846) and there's some code snippets that are working well on my sites for the time being. Not an acceptable solution in my opinion, but a serviceable one until Elementor does the right thing. Hopefully before my renewal next March, otherwise the solution is going to be to switch to [Bricks](https://bricksbuilder.io/) or something similar.

**Reply by awebguy99** [reply]
Completely agree, especially re the image optimization nags! I already have a plugin that does this. I tell my clients, don't worry, your images are as fast as possible...then they see these nag messages telling them they have unoptimized images!!!!

**Comment by Joni-L** [comment]
Not having this option for professional developers is sad

**Comment by daniilicic** [comment]
+1

**Comment by ghost** [comment]
It's absolutely insane that you keep putting us agencies through this. We could avoid issues like this if you allowed us to opt out of AI.  

== Changelog ==
= 3.22.1 - 2024-06-17 =
* Fix: Editor not loading due to compatibility issue between Elementor AI and third-party plugins

**Reply by ghost** [reply]
they just dont care about us.  Need to use other page builder.

**Comment by reedock** [comment]
+1

**Comment by squadcar** [comment]
This thread is obviously a great place to let the Elementor PR crew know your views. Another place is wordpress.org. Not in anyway saying you should leave a review, even if it would be useful for prospective customers. Either way, probably going to be more effective given their lack of response to the pro community in regards to this issue.

**Reply by awebguy99** [reply]
Good idea.

**Reply by ghost** [reply]
Good idea. Going for writing review and 1 star.  Will say it to other studios.

**Reply by ghost** [reply]
Great idea. Just posted my review. Encourage others here to do the same.

**Comment by awebguy99** [comment]
Honest question, does anyone know if the Brick builder does this? Upsell messages that we can't disable, even on a pro plan.

I really do love the Elementor builder **but this is getting silly, I've had multiple clients complain about this**.
I bought the Bricks lifetime some time ago, I may have to consider moving over.

**Reply by SolunaCG** [reply]
I switched to Bricks a few months ago and so far I´ve only created 2 client sites with Bricks but I haven´t found any upsell message anywhere in the builder. 

I also bought the Bricks lifetime a while ago and so far I´m very satisfied with what I got which is why I decided to create every new website with Bricks now and never use Elementor again. 
I used Elementor for years on many client sites but no matter how much I participated in threads here on github, the devs didn´t seem to care at all about fundamental things.

**Reply by awebguy99** [reply]
Thanks.

**Reply by ghost** [reply]
I seen Zion / Oxygen too.

**Reply by ghost** [reply]
As another commenter mentioned, Bricks does not upsell you like this. I created a video on YT where I walked through a comparison of E vs. Bricks when it comes to upselling, hijacking the WordPress Dashboard, etc. E should be embarrassed.

I too bought the lifetime license for Bricks and will use it for future sites. Not just because of the upsells, but because issues with components we can’t easily shut off (AI), literally cause the E editor to not load and that’s a nightmare for us agencies.

**Comment by c2wd** [comment]
Since Elementor has pushed more Elementor AI on us instead of listening and giving us the option of turning this garbage off, I have updated some snippets to help get rid of it for those that don't want it imposing on their workflow.

**Remove Elementor AI from the Editor:**
```
function remove_elementor_ai_script_from_editor() {
    if ( \Elementor\Plugin::$instance->editor->is_edit_mode() ) {
        wp_dequeue_script( 'elementor-ai' );
        wp_dequeue_script( 'elementor-ai-media-library' );
    }
}
add_action( 'elementor/editor/after_enqueue_scripts', 'remove_elementor_ai_script_from_editor' );
```

**Remove Elementor AI from the Media Library:**
```
function dequeue_elementor_ai_media_library_admin() {
    wp_dequeue_script('elementor-ai-media-library');
}
add_action('admin_enqueue_scripts', 'dequeue_elementor_ai_media_library_admin', 100);
```

**Reply by jamieburchell** [reply]
Unfortunately this doesn't remove "Generate with Elementor AI" from the "Featured image" and "Excerpt" areas of the WordPress editor. Maybe that's just been added.

For that,

`wp_dequeue_script( 'elementor-ai-gutenberg' );` appears to do the trick.

**Reply by c2wd** [reply]
I'll take a look. I have not updated to 3.23.0 yet because as we all know, it's always following by 3.23.1, 3.23.2, 3.23.3 within days.

**Reply by jamieburchell** [reply]
I found the hook (see edit above). I guess we could just find everything with `elementor-ai` and just dequeue it all. `elementor-ai-layout` doesn't need dequeuing because at least that only gets queued when the experiment is enabled (I think). Although the experiment probably won't work anyway with the other stuff dequeued.

**Reply by c2wd** [reply]
lol, this is so ridiculous we have to do all of this just to get rid of the AI. I'll try it all out later and update the snippets.

**Reply by jamieburchell** [reply]
Maybe we can get AI to remove the AI

**Comment by jamieburchell** [comment]
Can we opt out of this AI stuff now, please 🙏🏻

> 3.23.1 – 2024-07-15
>
> Fix: Slow loading of WordPress admin and Elementor screens **due to AI health check**


How about a global "agency" or "professional" mode where all the upsells are turned off too?

As an agency we already chose the solutions for our clients and they and we don't need to be advertised to.

**Reply by ghost** [reply]
This is really funny. 
They slowing our sites cause want money.. and forcing us to use AI. 
AI does not have any settings to be switched off. This is how they listening to us.

**Reply by mcrowell** [reply]
100% agree!

Elementor: What is this AI health check? And why is it enabled by default if we don't ever plan to use AI?

**Reply by jamieburchell** [reply]
> Elementor: What is this AI health check? 

It's called from a function called "get_usage" and it calls home to my.elementor.com/api/v2/ai/. Perhaps you opted-in to AI at some point @mcrowell? Maybe this only happens once accepting/acknowledging the AI beta screen.

**Reply by ghost** [reply]
I've alerted some major WP news outlets about Elementor AI. They need to be put on blast for overstepping. This has added soooo many headaches and extra work for my agencies and others.

**Comment by portrayaloflife** [comment]
@nicholaszein Just wanted to see if you could chime in here for all of us. Is anything being talked about or is anything going to be done to give Agencies an option to turn all the AI and notification popups off? It's negatively affecting our client experiences having them be sold to. An option for agencies to turn it off would be really appreciated.

**Reply by ghost** [reply]
Nothing is going to change, according to what I know so far. Even after following exactly the steps I was directed to take, it was confirmed to me that Elementor has no intention of changing this because it is how they advertise this feature.

This is the word from Elementor support, which was who I was directed to, to “file a complaint.”

I have a livestream with a major media outlet, on Tuesday, to discuss this issue. My hope is that enough people will speak up that it will facilitate change.

**Reply by portrayaloflife** [reply]
I would be fine even with Admins still seeing it and utilizing it, but let us turn it off for Editors, Authos etc

**Reply by ghost** [reply]
Then how would they advertise to those roles of your website? The whole point is to get every individual to sign up for their own Elementor AI account. Remember, it’s used on a Per-Account basis, not a Per-Website basis.

**Reply by portrayaloflife** [reply]
Make it a paid feature to remove

**Reply by awebguy99** [reply]
> Make it a paid feature to remove

I hope you're joking...

**Reply by ghost** [reply]
We’ve already paid to have it removed. It’s called ‘Elementor Pro.’

**Comment by borjamerino-v** [comment]
Please, I want that too. I hate the messages about AI and the absurd image reduction ads.

**Comment by c2wd** [comment]
Here is an updated code snippet to hopefully combat the latest round of AI and Image Optimization NAGS from Elementor. Feel free to give input and/or feedback so everyone can implement and get back to a nice clean workflow for ourselves and our clients.

```
function customize_elementor() {
    // Remove Elementor Nag Admin Notices
    echo '<style>.e-notice--cta { display: none !important; }</style>';

    // Remove Elementor AI script from editor
    if ( \Elementor\Plugin::$instance->editor->is_edit_mode() ) {
        wp_dequeue_script( 'elementor-ai' );
        wp_dequeue_script( 'elementor-ai-media-library' );
    }

    // Dequeue Elementor AI media library and Gutenberg scripts in admin
    wp_dequeue_script( 'elementor-ai-media-library' );
    wp_dequeue_script( 'elementor-ai-gutenberg' );
}

// Hook for removing Elementor nag in admin
add_action( 'admin_head', 'customize_elementor' );

// Hook for removing Elementor AI scripts from editor
add_action( 'elementor/editor/after_enqueue_scripts', 'customize_elementor' );

// Hook for dequeuing Elementor AI media library and Gutenberg scripts in admin
add_action('admin_enqueue_scripts', 'customize_elementor', 100);

```

**Reply by portrayaloflife** [reply]
Ooo Does this remove the Notification popup they throw up in the editor too for new users?

**Comment by teoeugene** [comment]
Version 3.23 has added even more AI insertion points into a Post's Featured Image and Excerpt fields for WordPress 6.5 and below. Tested on sites that still has not been updated with 3.23 and pretty sure this is a new addition:

![image](https://github.com/user-attachments/assets/44cfb9ac-056a-44a7-b996-79dd708175d5)

This does not appear for WordPress 6.6 as the layout for these fields has been updated.

**Comment by miriamelementor** [comment]
I just wanted to share an update: we have removed the majority of AI popups. We are also working on a way for users to easily disconnect their sites from AI. This functionality is in progress and we will update here as soon as it’s released.

**Reply by portrayaloflife** [reply]
THANK YOU!!!

**Reply by ghost** [reply]
Hope this will be real and soon. Hate this AI features.

**Comment by ghost** [comment]
[heart]         Ryan Logan reacted to your message:
________________________________

**Comment by LucasMariniFalbo** [comment]
I needed to disable the Optimized Control Loading experiment because messes with a core feature of my plugin. So I come up with this function that does this for me, avoiding the user to enable this experiment. 

Not sure if is usefull for you guys, but you can use it to avoid an experiment from being enabled. You can set it in your plugin or theme:

```
add_filter('elementor/experiments/default-features-registered', function ($manager) {
    
    // $manager returns an object 
    if ($manager instanceof \Elementor\Core\Experiments\Manager) {

        // The experiment values are private, so using Reflection allow us to take 'features' private properties
        $reflection = new \ReflectionClass($manager);
        $property = $reflection->getProperty('features');
        $property->setAccessible(true);
        $features = $property->getValue($manager);

        // Print the experiment names in case you need to check the name of the experiment you want to disable
        error_log( print_r($features, true) );

        // Check for an experiment and disable it (this example disables optimized control loading)
        if (isset($features['e_optimized_control_loading'])) {
            $features['e_optimized_control_loading']['state'] = 'inactive';
            $property->setValue($manager, $features);
        }
    }

    return $manager;
});
```

**Comment by mcrowell** [comment]
Why are the AI prompts now appearing in the block editor????

![ai-block](https://github.com/user-attachments/assets/f01ab1a4-50bb-4129-becc-8f4e886fddf7)

**Reply by robaxx** [reply]
Funnily enough, this was a suggestion I made to them some time ago. I said there was no value in AI editing if it's constrained to a single plugin. It would have to be in all parts of the editing experience to be useful. But I also said it needs to be able to be disabled entirely... so I guess we got one and not the other, for now.

**Comment by hitsgee** [comment]
I think they are listening 😄

Just spotted this PR https://github.com/elementor/elementor/pull/28303

**Reply by portrayaloflife** [reply]
This is awesome to see!

**Reply by m1ga** [reply]
let's see how long it takes for it the get merged. Some PRs are there for years and still open

**Reply by toineenzo** [reply]
> let's see how long it takes for it the get merged. Some PRs are there for years and still open

It has been committed! https://github.com/elementor/elementor/commit/ef52a9919ae20938856707e1171da38b8e50a7b4

**Reply by m1ga** [reply]
and merged ;-)

**Reply by toineenzo** [reply]
> and merged ;-)

Yeah, that's what i meant lol

**Reply by muzKore** [reply]
I think it's to late for me. I started using Elementor three years ago, got heavily involved on the technical side but this kind of behaviour is ridiculous.  As a comparison,  I've been checking out Bricks. They actually listen and add features that make a difference. I've been building a site and it takes a bit to get used to but it actually makes so much more sense. The code is clean, better ACF integration and inline dynamic data already plugged in. I'm actually shocked and it is also so much faster fast. 
Will keep managing my Elementor sites and I'll be building more with bricks in the future.

**Comment by ghost** [comment]
Please do sitewide diable.. not only per user.

**Reply by aksellno** [reply]
+1 If not for the Free version, then at least in the Pro version this should be possible.

**Reply by ghost** [reply]
@miriamelementor when developers will do exactly same what customers want?)) 

Need to do brainstorm again.
If customer NOT need AI, he will NOT use this.. not today. not tomorrow. 
IF Admin not use AI, his users will NOT USE TOO. 

Do not make your customers angry.  Just make option - DISABLE FOR ALL.

**Reply by awebguy99** [reply]
I haven't had a chance to try this out yet. Are you saying there isn't now an option to completely disable all AI (including popup and link ads), sitewide? You have to it per user?!

**Reply by black-eye** [reply]
Yes please, give us an option to disable AI (and the other stuff), completely and globally for all the users (and everywhere). One cannot avoid the feeling, that when the developers were forced to add this option, at least they did it the least user friendly way, hoping that the most of the users will resign to do the extra work and will leave it as it is.

For example, here is one of my use cases - multisite with 5 subsites, using the same custom theme. On each subsite there are lower tens of users, most of them editors with lower capabilities. Which makes it cca 100 edits to make this AI feature go away (instead of 5, or 1 using the filter in the theme). 

Will I make an effort to do all this extra work? No, I'll keep using the hacky way showed here: https://github.com/orgs/elementor/discussions/22796#discussioncomment-10141694. Thus, this new feature is of no use for me... :(

I completely agree with the opinion presented in this video: https://www.youtube.com/watch?v=BsCkh3kjjyQ 
I'm so frustrated with Elementor lately... Why easy tasks cannot be solved easily?

**Comment by awebguy99** [comment]
OK, I'm amazed. Just check out this video!!
https://www.youtube.com/watch?v=BsCkh3kjjyQ

Elementor, what are you doing?!!!! You finally give us the option to disable (the unwanted, not asked for) AI features, **only to force us to disable it PER USER. What about sites with hundreds or thousands of users?**

I love using Elementor and have done a number of feature requests and bug reports to improve the product but this might be the final straw to push me onto using Bricks.

**Reply by ghost** [reply]
Yes, looks like they just trolling us. 

- "Good luck to disable for every user our AI best services"

And if you have +100 users in every day = you must disable for all in MANUAL mode. for every user. every day.

What you are doing? Why you hate us so much? What words you not understand from phrase "Give us posibility to DISABLE IT completely"? 

We are not writed "to disable per-user".
WE ARE NOT want to see this AI. 

Even if AI will create best site from blank page, even if this will be BEST optimizer for images, even if AI will write best code / text. 
For most of us - this is UNWANTED function. 
But you force us to use it, or to disable this per user.. to spend hours to disable it every day.

I think... need to find alternative to Elementor. sad but.. true, and to switch all sites from elementor to another builder.

**Reply by onotype** [reply]
I don't get how anyone would look at this request/thread and thought "a site admin would want to disable this on their account and leave it on for all their users/clients. Yep, that's what they want here, definitely."

The funny thing is that instead of just adding a simple toggle that does what everyone wants, they really went out of their way to make a toggle in the Elementor Settings page that does nothing of value, AND THEN added a toggle in the user settings, where they have to store the status for each user in the db. This wasn't oversight or incompetence.

Let's also not get lost in the sauce: this isn't a pro vs free feature. I doubt they'll add "Pro Benefits: able to remove the AI we added ourselves, site wide." Otherwise, they'll have to admit or somehow acknowledge their goals with this AI doesn't have any direction. But at this point, I wouldn't put it past them.

**Comment by robaxx** [comment]
Does someone maybe know how to make a function to disable it for all users, given that there is obviously a switch per user in there somewhere?

**Reply by ghost** [reply]
Using some plugins.. or disable via code / css..  they dont care about us.. so no way to disable officially in Elementor.

**Comment by babsg1983** [comment]
I need to be able to disable Elementor AI programmatically not manually per user as I have 1000's of subscribers in a WP/Learndash Instance. Any suggestions would be appreciated. I was instructed by the Elementor Tech advisor to post my concerns here. I want to choose whether or not I use. Bit concerned regarding the ability to control the resources I use.
